README.md

```diff
@@ -80,3 +82,54 @@ $ npm install
 $ npm run clean
 $ npm run build
 ```
+
+## For server-side rendering via node.js
+
+Copy `xypic.js` to your npm project, and create a `mathjax.js` in the same directory containing:
+
+```js
+import { readFileSync, writeFileSync } from 'fs';
+import { argv } from 'yargs';
```

Comment:
Is it really necessary to use `yargs`?  you are not really using any of its functionality, and could really just use `process.argv` (removing the program name first).

Reply:
I think it's better to use `yargs`, because the user might want to run `npm run --silent` or some such thing, which complicates things for `process.argv`.